website_lazy_load_image/models/ir_ui_view.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from lxml import etree
+
+
+class IrUiView(models.Model):
+    _inherit = 'ir.ui.view'
+
+    DEFAULT_SRC = 'data:image/gif;base64,R0lGODlhAQABAIAAAP///' \
+                  'wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=='
+
+    @api.model
+    def render_template(self, template, values=None, engine='ir.qweb'):
```

Comment:
docstring documenting why and what we do

Reply:
weird... AFAIR `hasclass` is peculiar to odoo's inheritance rendering... :thinking: 
well, TIL :wink: 
@tarteo thanks! Small thing to change in the test but looks good.